inc/load.php

```diff
@@ -118,6 +118,16 @@ function load_autoload($name){
         return true;
     }
 
+    // plugin namespace
+    if(substr($name, 0, 7) == 'plugin/') {
+        $name = str_replace('/test/', '/_test/', $name); // no underscore in test namespace
```

Comment:
Not the physical folder is intended to be renamed, but the namespace ```plugin/myplugin/test/something/``` is intended to be mapped to the folder ```plugin/myplugin/_test/something/```

However now that I'm reading about [standards concerning php namespacing](http://www.php-fig.org/psr/psr-4/), I'm not so sure that this is the most portable approach.
On the other hand dokuwiki plugin are never going to be ported to or from anywhere else anyway, so maybe these standards do not apply to us at all?